nim/templates/index.html

```diff
@@ -29,27 +29,43 @@ <h1>So it begins...</h1>
 
     <div id=piles></div>
 
-
     <div style="clear: both;">
       <button onclick="updateGameState()">Make Move</button>
+    </div>
+
+    <div>
+      <input type="number" id="pile" value="1" min="1">
+      pile
+    <div>
+    </div>
+      <input type="number" id="stones" value="1" min="1">
+      stones
+    </div>
+
+    <div>
       <button onclick="newGame()">New Game</button>
     </div>
 
     <div>
-      <input type="number" id="numPiles" value="3" min="1">
+      <input type="number" id="numPiles" value="3" min="1" onchange="newGame()">
       total piles
     </div>
 
     <div>
-      <input type="number" id="minPerPile" value="4" min="1">
+      <input type="number" id="minPerPile" value="4" min="1" onchange="newGame()">
       minimum stones per pile
     </div>
 
     <div>
-      <input type="number" id="maxPerPile" value="12" min="1">
+      <input type="number" id="maxPerPile" value="12" min="1" onchange="newGame()">
       maximum stones per pile
     </div>
 
+    <div>
+      <input type="checkbox" id="delay">
+      delay for bot move
```

Comment:
Err ok, so it just makes our game slower? Why?

Reply:
Pushing refactor. No longer applicable.